src/main/java/gregtech/common/metatileentities/multi/electric/generator/TurbineMultiblockController.java

```diff
@@ -0,0 +1,57 @@
+package gregtech.common.metatileentities.multi.electric.generator;
+
+import gregtech.api.metatileentity.multiblock.MultiblockAbility;
+import gregtech.api.recipes.machines.FuelRecipeMap;
+import gregtech.common.metatileentities.electric.multiblockpart.MetaTileEntityRotorHolder;
+import net.minecraft.util.ResourceLocation;
+
+import java.util.List;
+
+public abstract class TurbineMultiblockController extends FueledMultiblockController {
```

Comment:
In this case more universal name would be better as it does not have to be tied with Turbine anymore.